.sync.yml

```diff
@@ -13,6 +13,7 @@ Gemfile:
         version: '~>2.14.1'
       - gem: puppet-lint
       - gem: puppetlabs_spec_helper
+        version: '~>0.10.3'
```

Comment:
If we require at greater than 0.9 than why not do ~> 0.10? is there a crucial bugfix in 0.10.3?